src/coreclr/vm/jitinterface.h

```diff
@@ -238,15 +238,10 @@ extern "C" FCDECL2(Object*, ChkCastAny_NoCacheLookup, CORINFO_CLASS_HANDLE type,
 extern "C" FCDECL2(Object*, IsInstanceOfAny_NoCacheLookup, CORINFO_CLASS_HANDLE type, Object* obj);
 extern "C" FCDECL2(LPVOID, Unbox_Helper, CORINFO_CLASS_HANDLE type, Object* obj);
 
-#if defined(TARGET_ARM64) || defined(FEATURE_WRITEBARRIER_COPY)
```

Comment:
There is `ETW::MethodLog::StubsInitialized` method that gives names to the adhoc dynamically generated helpers in sampling traces so that they do not show up as broken stacks. We may want to start reporting the write barriers using this event.

Reply:
That's nice, I'll add it.